.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -31,9 +33,10 @@ assignees: ""
 - Version: [e.g. 22, latest]
 - Device / System: [e.g. PC, iPhone 6]
 
-## Screenshots
+### Screenshots
 
 <!-- Help us understand the problem with screenshots. You can use [peek](https://github.com/phw/peek) to generate animated GIFs. -->
+<!-- Take care of not revealing sensitive data in those screenshots/videos. -->
```

Comment:
Maybe I would add something like "WARNING:" or "NOTICE:" in front of this sentence.